app/settings/globals.py

```diff
@@ -81,3 +80,22 @@
     port=WRITER_PORT,
     database=WRITER_DBNAME,
 )
+
+AWS_REGION = config("AWS_REGION", cast=str, default="us-east-1")
+
+POSTGRESQL_CLIENT_JOB_DEFINITION = config(
```

Comment:
should these have default values like this? If they're accidentally missing from the environment, we won't know until we get an error for trying to submit a batch job with a non-existent queue or job definition.